src/transactions/types.ts

```diff
@@ -34,7 +34,16 @@ export type CompletedStandbyTransaction = (
   context: TransactionContext
 }
 
+export type FailedStandbyTransaction = (
```

Comment:
lol naming is hard 😅 i agree, i think this is okay. it would be misleading if we had "success" in the name but completed and confirmed both seem correct to me